lib/theme/colors.js

```diff
@@ -34,8 +34,8 @@ export const colors = {
   notice: '#EB7720',
   noticeAlt: '#FFFBEF',
   onNotice: '#EB7720',
-  warning: '#CB532D',
-  warningAlt: '#F75524',
+  warningAlt: '#CB532D',
+  warning: '#F75524',
```

Comment:
What's the reason for swapping these? We should make sure that other components aren't affected.

Reply:
i didn't see any other warningAlt being used